report_wkhtmltopdf_param/models/report.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Avoin.Systems
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class Report(models.Model):
+    _inherit = 'report'
+
+    def _build_wkhtmltopdf_args(self, paperformat,
+                                specific_paperformat_args=None):
+        # noinspection PyUnresolvedReferences,PyProtectedMember
```

Comment:
This comment should not be removed. It is used by pyCharm to ignore the warning messages. IMHO is very beneficial.